Script/define.dat

```diff
@@ -631,6 +640,15 @@
 #define BRAVERY 1
 #define FEAR 2
 #define CONFUSION 3
+#define HEALING 4
+#define PLENTY 5
+
+#define POTTED_CACTUS 1
+#define POTTED_PLANT 2
+#define DEAD_FISH 3
+#define BONE_FISH 4
```

Comment:
I think dead_fish should turn into bone_fish when it's eaten

Reply:
Maybe you swallow the bones as well? :P

I'll try to do that, though. :) I like it.